pyincore/models/hazard/tornado.py

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2019 University of Illinois and others. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License v2.0 which accompanies this distribution,
+# and is available at https://www.mozilla.org/en-US/MPL/2.0/
+from pyincore import HazardService, Dataset
+from pyincore.models.hazard.hazard import Hazard
+from pyincore.models.hazard.hazardDataset import TornadoDataset
+from pyincore.models.units import Units
+
+from shapely.geometry import Point
+import random
+
+
+class Tornado(Hazard):
+    """
+    "tornadoType": "dataset",
+    "id": "xxx",
+    "name": "pytest - Joplin Tornado",
+    "description": "Joplin tornado hazard",
+    "creator": "xxx",
+    "threshold": null,
+    "thresholdUnit": "mph",
+    "spaces": [
+        "xxx"
+    ],
+    "date": "2020-08-14T16:22:32+0000",
+    "datasetId": "xxx"
+    """
+
+    def __init__(self, metadata, ef_rating_field="ef_rating", ef_wind_speed=(65, 86, 111, 136, 166, 200),
+                 max_wind_speed=250.0, seed=1234):
```

Comment:
I think seed should be -1 here and we need to check tornadoParameters object to see if a seed value was set.